ts/src/base/Exchange.ts

```diff
@@ -1055,7 +1059,11 @@ export default class Exchange {
                  (sinceIsDefined ? (entry[key] >= since) : true)))
         }
         if (limit !== undefined && limit !== null) {
-            array = tail ? array.slice (-limit) : array.slice (0, limit)
+            const arrayLength = array.length;
+            if (arrayLength > 0) {
```

Comment:
Comparing two values so you need to make sure length is more than 1
```suggestion
            if (arrayLength > 1) {
```

Reply:
It actually doesn't matter, if there's only one value then your just comparing it to itself, it would be useless, but nothing will break

Reply:
true

Reply:
> true

I guess it's some slight efficiency